int_tests/sb_int_test.go

```diff
@@ -13,28 +12,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func setUpCommand(dir, name string, arg ...string) {
+const (
+	HOME = "/home/test"
+)
+
+func command(t *testing.T, dir, name string, arg ...string) (out []byte) {
 	cmd := exec.Command(name, arg...)
 	cmd.Dir = dir
 	out, err := cmd.CombinedOutput()
 	if err != nil {
-		log.Fatal(string(out))
-	}
-}
-
-func testWriteFile(filename string, data []byte, perm os.FileMode, t *testing.T) {
-	err := ioutil.WriteFile(filename, data, perm)
-	if err != nil {
-		t.Fatal(err)
+		t.Fatal(string(out))
```

Comment:
I think you want to print the err here

Reply:
Yep yep, I do: https://golang.org/pkg/testing/#T.Fatal